notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb

```diff
@@ -1085,11 +1038,11 @@
         "id": "7421c559"
       },
       "source": [
-        "### Model Selection\n",
+        "### Model selection\n",
         "\n",
-        "Now that we have evaluated the models independently, we are going to move forward with only one of them. This election will be done based on the model evaluation metrics gathered in the previous steps.\n",
+        "After the models are evaluated independently, you're going to move forward with only one of them. The selection is done based on the model evaluation metrics gathered in the previous steps. The selected model is then deployed to an endpoint.\n",
```

Comment:
- change: "After the models are evaluated independently, you're going to move forward with only one of them"
    - "After the models are evaluated independently, you're going to only move forward with one of them."